public/templates/agents/agents-fim.html

```diff
@@ -1,72 +1,96 @@
                 </md-card-content>
             </md-card>
-       </div>
-
+        </div>
+    </div>
 
+    <div layout="row" class="height-230">
+        <md-card flex="33" class="wz-md-card">
+            <md-card-content class="wazuh-column">
+                <span class="wz-headline-title">Files added</span>
+                <md-divider class="wz-margin-top-10"></md-divider>
+                <kbn-vis id="Wazuh-App-Agents-FIM-Files-added" vis-id="'Wazuh-App-Agents-FIM-Files-added'"></kbn-vis>
+            </md-card-content>
+        </md-card>
+        <md-card flex="33" class="wz-md-card">
+            <md-card-content class="wazuh-column">
+                <span class="wz-headline-title">Files modified</span>
+                <md-divider class="wz-margin-top-10"></md-divider>
+                <kbn-vis id="Wazuh-App-Agents-FIM-Files-modified" vis-id="'Wazuh-App-Agents-FIM-Files-modified'"></kbn-vis>
+            </md-card-content>
+        </md-card>
+        <md-card flex="33" class="wz-md-card">
+            <md-card-content class="wazuh-column">
+                <span class="wz-headline-title">Files deleted</span>
+                <md-divider class="wz-margin-top-10"></md-divider>
+                <kbn-vis id="Wazuh-App-Agents-FIM-Files-deleted" vis-id="'Wazuh-App-Agents-FIM-Files-deleted'"></kbn-vis>
+            </md-card-content>
+        </md-card>
+    </div>
 
+    <div layout="row" layout-align="space-between stretch" class="height-570">
+        <md-card flex class="wz-md-card">
+            <md-card-content class="wazuh-column">
+                <span class="wz-headline-title">Alerts summary</span>
+                <md-divider class="wz-margin-top-10"></md-divider>
+                <kbn-vis vis-id="'Wazuh-App-Agents-FIM-Alerts-summary'"></kbn-vis>
+            </md-card-content>
+        </md-card>
+    </div>
 </md-content>
+
+<div layout="row" class="wz-padding-top-0">
+    <md-card flex="" class="wz-md-card _md flex">
+        <md-card-content>
+            <span class="wz-headline-title"><i class="fa fa-fw fa-file"></i> Syscheck files</span>
+            <md-divider class="wz-margin-top-10"></md-divider>
+            <div layout="row" class="wz-margin-top-10">
+                <input placeholder="Filter files..." ng-model="filesSearch" type="text" class="kuiLocalSearchInput ng-empty ng-pristine ng-scope ng-touched ng-valid height-30"
+                    aria-invalid="false" wz-enter="searchSyscheckFile(filesSearch)">
+                <button type="submit" aria-label="Search" class="kuiLocalSearchButton height-30" ng-click="searchSyscheckFile(filesSearch)">
+                    <span class="fa fa-search" aria-hidden="true"></span>
+                </button>
+            </div>
+            <div layout="row" ng-if="agent && !load" class="wz-margin-top-10 wz-margin-bottom-40-inv">
+                <wz-table flex path="'/syscheck/' + agent.id" row-sizes="[10,8,6,3]" extra-limit="true" keys="['file','size', 'perm', 'date']">
```

Comment:
`file`, `size`, `gname`, `uname`, `perm`. If it fits, add `mdate` too. Also remember to add their translations to the proper file.